generators/base/generator-base-private.mjs

```diff
@@ -162,29 +158,8 @@ export default class PrivateBase extends Generator {
    * @param folder
    */
   removeFolder(folder) {
-    folder = this.destinationPath(folder);
-    if (folder && shelljs.test('-d', folder)) {
-      this.log(`Removing the folder - ${folder}`);
-      shelljs.rm('-rf', folder);
-    }
-  }
-
-  /**
-   * @private
-   * Execute a git mv.
-   *
-   * @param {string} source
-   * @param {string} dest
-   * @returns {boolean} true if success; false otherwise
-   */
-  gitMove(source, dest) {
-    source = this.destinationPath(source);
-    dest = this.destinationPath(dest);
-    if (source && dest && shelljs.test('-f', source)) {
-      this.info(`Renaming the file - ${source} to ${dest}`);
-      return !shelljs.exec(`git mv -f ${source} ${dest}`).code;
-    }
-    return true;
+    // TODO Should not update variable
+    folder = deleteFolder(this, folder);
```

Comment:
```suggestion
    file = this.destinationPath(file);
    this.log(`Removing the folder - ${folder}`);
    rmSync(file, { force: true, recursive: true });
```

Reply:
Not agree, extraction rule should be straightforward

Reply:
taken into account